### PR TITLE
Improves builder developer experience in docs

### PR DESCRIPTION
**Pull request by ckartik**
* Moves searcher testing docs to separate searcher.md file
* Provides clearer build instructions

### CODE REVIEW

**Comment by floodcode** [comment]
Looks good! But i would remove mentions of rollup.json as we no longer use it

File: docs/searcher-testing.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+4. Once you have created your wallet, you will need to add the sepolia network to your wallet. To do this, click on the network dropdown and you should see show/hide Testnets. Hit the link, and toggle testnets. You should now see sepolia and other testnets in the network dropdown.
+5. Now go ahead and click the profile picture circle on the top right of your wallet interface and hit create account and name it "searcher emulator".
+6. Now switch back to account 1 and hit account details and rename it "builder". Now click export private key from the account details page and copy the private key. You will need this later.
+7. Connect to the searcher account in your wallet and get some sepolia ETH from the faucet [here](https://sepolia-faucet.pk910.de/). Once you've acquired enough seploia eth, send a transaction with 5-10% of the sepolia eth to the Builder Account. You can get the builder account address that you will need from the wallet, it will appear representation, similar to the following: 0x9f2...5E40
+
+A contract is currently deployed here:
+- https://sepolia.etherscan.io/address/0xc38e581D0403b4065F4d61A838431B143ceE4c81
```
Comment (floodcode) [inline docs/searcher-testing.md:20]: Contact address was changed recently, take it from boos/main.go

File: docs/searcher-testing.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+```
+
+**Step 2.** Run `boost` command:
+```
+$ ./boost --addr :8080 --rollupkey `<builder-private-key>` --buildertoken `<builder-auth-token>`
+```
+You can get the builder private key from step 6 in the wallet setup section.
+You come up with you're own builder auth token, you will want to set this value as a header `X-BUILDER-TOKEN` in your Builder instance while posting data to Boost.
+
+**Step 3.** Go [here](https://sepolia.etherscan.io/address/0xc38e581D0403b4065F4d61A838431B143ceE4c81#writeContract) and click on the `setMinimalStake`. Here you can connect the wallet associated with the builder and set the minimal stake needed to connect to you to get execution hints.
```
Comment (floodcode) [inline docs/searcher-testing.md:34]: Function name changed to updateBuilder()

Comment (ckartik) [reply docs/searcher-testing.md]: Done!

**Review by floodcode — APPROVED** [review]

**Pull request merged**